cea/technologies/chiller_vapor_compression.py

```diff
@@ -77,22 +77,35 @@ def calc_COP(T_cw_in_K, T_chw_re_K, q_chw_load_Wh):
     return COP
 
 
-def calc_COP_with_carnot_efficiency(peak_cooling_load, q_chw_load_Wh, T_chw_sup_K, T_cw_in_K, VC_chiller):
+def calc_COP_g(T_evap_K, T_cond_K, VC_chiller):
     """
-    Calculate the weighted average part load factor across all chillers based on the load distribution and derive the
-    COP based on that. This way of calculating the chiller COP resembles the approach described in EN 14511-2:2018 and
-    EN 14825:2018. The latter is illustrated in [P. Conti et al., 2020]
-
-    ..[P. Conti et al., 2020] P. Conti, C. Bartoli, A. Franco and D. Testi (2020). Experimental Analysis of an Air Heat
-    Pump for Heating Service Using a “Hardware-In-The-Loop” System
-    TODO: Continue looking for a better source or change the formula (possibly check ASHRAE Standard 90.1-2001 on
-          chiller rated efficiency if available)
+    Calculate the approximate COP at rated operating conditions using the g-value (sometimes also called
+    the second-law efficiency [Bejan, 2016]).
+    Assuming a rated COP for all calculations is a strong simplification, but accurate enough for most cases in CEA.
+
+    [Bejan, 2016] Adrian Bejan, 2016, Andvanced engineering thermodynamics (p. 106)
     """
-    PLF = calc_averaged_PLF(peak_cooling_load, q_chw_load_Wh, T_chw_sup_K, T_cw_in_K, VC_chiller)
```

Comment:
Do you know if this function will still be used somewhere in CEA? If not, isn't it better to remove it?

Reply:
This function is still used in one of the unit-tests. But is no will not in any part of the CEA scripts. I've been contemplating what to do about functions like this. The function is a valid implementation of what is presented on comnet.org (which is a partial replica of the ASHRAE-norms on cooling systems). So my thinking is it might come in handy at some other point in time.

Is there maybe some sort of a repository, where we could move functions like this to while they are not being used in the main code of CEA? Or can we create one?

Reply:
@shanshanhsieh @reyery I remember there was some legacy code in CEA for cases like this. Should we still follow this approach for cases like this?

Reply:
After asking the question on the developers team slack channel, the decision was to delete the unused functions of the script.